### ISSUE TITLE
After sign-in to GitLab in Firefox, an empty HTML file is downloaded automatically

### ISSUE THREAD

**Issue by schoettl** [200ok-ch/organice#816]
**Describe the bug**
about:blank

**To Reproduce**

1. Sign out
2. Sign in using your GitLab project
3. Authorize access on the gitlab page
4. On redirect to the app view, Firefox downloads an empty HTML file wtih a random filename like NrTOQHJ2.html

But I couldn't reproduce it a second time...

**Expected behavior**

Shouldn't download a file.

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox

(Could not reproduce on Android with Firefox.)

**Comment by munen** [comment]
Hi @schoettl 

Thank you for the report! I just tested on Safari in iOS and logged in and out twice without reproducing. I’ll try soon with FF on Linux. 

At least it’s good that it can still work, since Android and iOS still function^^

Good night and speak soon👋

**Comment by munen** [comment]
I tried on FF 100 on Debian Linux and couldn't reproduce it. However, I also do not get the Gitlab authorization screen anymore. I guess everyone sees that one only once. I go directly from the /sign_in screen to being logged in.

So this downloading of the HTML file might be an edge case that only happens for the first login. @schoettl Do you remember if you've actually been logged in on your first try, or did you only get the HTML file as a response and had to try a second time?

**Comment by schoettl** [comment]
@munen Yes, I got this download only the very first time, I signed in with GitLab (after the authorization step)! So it's hard to reproduce...
